packages/calcite-components/src/components/text-area/text-area.tsx

```diff
@@ -426,34 +430,61 @@ export class TextArea
     };
   }
 
-  private replacePlaceholdersInMessages(): string {
-    return this.messages.tooLong
-      .replace("{maxLength}", this.localizedCharacterLengthObj.maxLength)
-      .replace("{currentLength}", this.localizedCharacterLengthObj.currentLength);
+  private replacePlaceholders(
```

Comment:
nitpick: can we retain the name `replacePlaceholdersInMessages()` . Helps providing context of placeHolders and avoid confusing with `place-holder` property of text-area.